_posts/en/newsletters/2020-07-29-newsletter.md

```diff
@@ -0,0 +1,236 @@
+    need to pause sending or relaying payments in a particular channel
+    in order to upgrade it.
+
+    ZmnSCPxj [noted][zmnscpxj re-funding] that the same basic idea could
+    be used to essentially update the funding transaction offchain, such
+    as the case where [taproot][topic taproot]
+    and [SIGHASH_ANYPREVOUT][topic sighash_noinput] are implemented,
+    allowing [Eltoo][topic eltoo]-based channel commitments to be used.  In
+    ZmnSCPxj's proposal, the output of the existing funding transaction
+    would be paid to a new funding transaction that is kept offchain.  If
+    the channel terminates with a mutual close, the original funding
+    transaction output is paid to the final channel balances; otherwise,
+    the offchain secondary funding transaction can be published onchain
+    and the channel can be resolved using the appropriate unilateral
+    close protocol.
+
+## Field report: Using Descriptors and PSBT at River
+
+{% include articles/river-descriptors-psbt.md %}
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What are "leaf versions" in Taproot?]({{bse}}97104)
+  Michael Folkson and Pieter Wuille explain that each leaf node of a
+  [taproot][topic taproot] tree commits to both a leaf version and a script.
+  This leaf version can be used for upgradability and designates the script
+  semantics that apply just to that leaf.
+
+- [What are the different upgradability features in the BIP-Taproot (BIP341) proposal?]({{bse}}96951)
+  Michael Folkson answers a question from Twitter regarding the different ways
+  in which taproot can enable upgradability including leaf versions for script
+  semantics, repurposing opcodes for future functionality, pubkey types, and the
+  annex for new fields.
+
+- [Is there an active list of BIPs currently open?]({{bse}}97043)
+  Pieter Wuille describes various methods that have previously been used to
+  activate soft forks and notes that, while there are currently no unactivated
+  soft forks in Bitcoin Core, there is a discussion about the activation method
+  for taproot. Pieter also answers a [similar question][stack exchange miner
+  signaling] about mining signaling support.
+
+- [Could we skip the Taproot soft fork and instead and use Simplicity to write the equivalent of Taproot scripts?]({{bse}}97049)
+  Michael Folkson, quoting Pieter Wuille, outlines the current state of
+  [Simplicity][news96 simplicity] noting also that integrating Simplicity in
```

Comment:
ISTM there is a missing comma before "noting", or alternatively s/noting also/and also notes/